pkgs/by-name/bi/bitwarden-directory-connector/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  pkgs,
+  jq,
+  python3,
+  pkg-config,
+  libsecret,
+  nodejs_18,
+}: let
+  buildNpmPackage' = buildNpmPackage.override {nodejs = nodejs_18;};
+in
```

Comment:
It looks like recent [changes](https://github.com/NixOS/nixpkgs/commit/5aaeafbe2603df195938edaa91c2a68739eb499d) to `buildNpmPackage` make this unnecessary. You can provide a `nodejs` attribute in the main argument attrset instead.

Reply:
This works really well and makes it cleaner :D